ehrapy/preprocessing/_quality_control.py

```diff
@@ -263,13 +263,13 @@ def qc_lab_measurements(
         adata: Annotated data matrix.
         reference_table: A custom DataFrame with reference values. Defaults to the laposata table if not specified.
         measurements: A list of measurements to check.
-        unit: The unit of the measurements. Defaults to SI .
+        unit: The unit of the measurements. Defaults to 'traditional'.
```

Comment:
Hmm, should this be the default? Is it really more common?

Reply:
And is it true? :joy:

Reply:
I changed it to `traditional` due to this line: 

`reference_column = "SI Reference Interval" if unit == "SI" else "Traditional Reference Interval"`

But I don't really know what is real default